src/measure_theory/measure/measure_space.lean

```diff
@@ -604,6 +604,14 @@ else 0
   map f μ s = μ (f ⁻¹' s) :=
 by simp [map, dif_pos hf, hs]
 
+lemma to_outer_measure_map {f : α → β} (hf : measurable f) :
```

Comment:
```suggestion
lemma map_to_outer_measure {f : α → β} (hf : measurable f) :
```
I believe in other lemmas `to_outer_measure` is consistently named as if it were a postfix.

bors d+